taxonomy-editor-frontend/src/pages/editentry/ListTranslations.jsx

```diff
@@ -70,13 +69,13 @@ const ListTranslations = ({ nodeObject, setNodeObject }) => {
 
   const handleDialogConfirm = (newShownLanguageCodes) => {
     newShownLanguageCodes.forEach((languageCode) => {
-      if (
-        !renderedTranslations.some((item) => item.languageCode === languageCode)
-      ) {
-        handleAddTranslation(languageCode);
-      }
+      handleAddTranslation(languageCode);
     });
     setShownLanguages(newShownLanguageCodes);
+    localStorage.setItem(
+      "shownLanguages",
```

Comment:
this key should live outside of the component (be declared before the component...), `const SHOWN_LANGUAGES_KEY`